### PR TITLE
Added the Space Cadet Jumpsuit, a new star trek inspired jumpsuit

### PR DESCRIPTION
**Pull request by honeyed-lemons**
:cl:
- add: Added the Space Cadet Jumpsuit in all jumpsuit colors

### CODE REVIEW

File: Resources/Prototypes/Loadouts/Jobs/Civilian/passenger.yml [hunk 1]
```diff
@@ -57,6 +57,15 @@
   equipment:
     jumpsuit: ClothingUniformJumpsuitAncient
 
+# The Next Generation
```
Comment (hivehum) [inline Resources/Prototypes/Loadouts/Jobs/Civilian/passenger.yml:60]: this should be in our file structure and not editing an existing file! make a Resources/Prototypes/_Impstation/Loadouts/Jobs/Civilian/passenger.yml and move this entity there

**Review by hivehum — CHANGES_REQUESTED** [review]
i love this btw. please make colored versions for other departments. (in a different pull request this can just be the passenger outfit)

File: Resources/Prototypes/Loadouts/loadout_groups.yml [hunk 2]
```diff
@@ -304,6 +304,7 @@
   - GreyJumpskirt
   - AncientJumpsuit
   - RainbowJumpsuit
+  - TNGJumpsuit
```
Comment (hivehum) [inline Resources/Prototypes/Loadouts/loadout_groups.yml:307]: you changed the name of the greytider jumpsuit without changing it here

Comment (honeyed-lemons) [reply Resources/Prototypes/Loadouts/loadout_groups.yml:307]: WAHH NOoo

**Review by hivehum — APPROVED** [review]
looks good!!

**Pull request merged**